### PR TITLE
dispute remaining & total stake

### PR DESCRIPTION
**Pull request by epheph**

### CODE REVIEW

File: src/server/getters/get-dispute-info.ts [hunk 1]
```diff
@@ -138,6 +146,7 @@ function reshapeStakeRowToUIStakeInfo(stakeRows: DisputesResult): UIStakeInfo|nu
   const disputeRound = totalCompletedStakeOnAllPayouts.equals(0) ? null : stakeRows.disputeRound[0] == null ? 0 : stakeRows.disputeRound[0].disputeRound;
   return {
     marketId: marketRow.marketId,
+    stakeCompletedTotal: totalCompletedStakeOnAllPayouts.toFixed(),
```
Comment (bthaile) [inline src/server/getters/get-dispute-info.ts]: This is the total stake for the last completed dispute bond? Or is it the stake needed to complete the current active disptue?

Comment (epheph) [reply src/server/getters/get-dispute-info.ts]: It is the sum of all currently completed stake, including initial, so you will need to double it. I could double add a double'd version if you like.

Comment (bthaile) [reply src/server/getters/get-dispute-info.ts]: Could you add a stakeCurrentTotal that would be the current dispute bond total? Does that make sense? 
stakeCurrentTotal would be double the stakeCompletedTotal. So if the market is in awaiting fee window after designated reporter then stakeCompletedTotal would be the DR stake and stakeCurrentTotal would be double stakeCompletedTotal, correct?

**Comment by bthaile** [comment]
I'll run with your current changes. Also is the BigNumber string issue been merged up? I see some of the toString() have been removed.

**Comment by epheph** [comment]
accidental removal of toString, i'll re-add

**Comment by epheph** [comment]
I think we can do without the ones i calculatebondsize, since those have to be bignumber's already

**Pull request merged**